src/types/weight_proof.py

```diff
@@ -80,6 +80,13 @@ class SubEpochSegments(Streamable):
     challenge_segments: List[SubEpochChallengeSegment]
 
 
+@dataclass(frozen=True)
+@streamable
+# this is used only for serialization to database
+class RecentChainData(Streamable):
```

Comment:
If we are changing the DB format won't that break old nodes that have the old format?

Reply:
bad commet ill fix, used for serialization to the multi process

Reply:
dident change any of the dbs structure